### ISSUE TITLE
[OMEMO] Keep getting the message "Encryption is disabled" even though it is enabled and works.

### ISSUE THREAD

**Issue by SharkyRawr** [psi-im/plugins#92]
The plugin keeps showing notices in the chat, alternating between `OMEMO encryption is enabled` when I receive a message and `Encryption is disabled` when I send a message. Keys are set-up and trusted and I have no idea why this happens. The other party confirmed the message was encrypted.

**Comment by Ri0n** [comment]
usually it happens on receiving of unencrypted messages iirc

**Comment by Neustradamus** [comment]
I have already informed about this problem.

**Comment by SharkyRawr** [comment]
> I have already informed about this problem.

Ah good, what's the issue id?

**Comment by Neustradamus** [comment]
@SharkyRawr: I am happy that you confirm one OMEMO bug, I am not alone to create tickets :)

I have informed @tehnick by several tickets and in the MUC Room ^^

Some tickets:
- https://github.com/psi-im/psi/issues?q=is%3Aissue+sort%3Aupdated-desc+omemo+
- https://github.com/psi-im/plugins/issues?q=is%3Aissue+sort%3Aupdated-desc+omemo+

**Comment by Neustradamus** [comment]
@kssytsrk: Can you look this ticket?

**Comment by kssytsrk** [comment]
shouldn't this be closed as duplicate of psi-im/psi#616? Or that one closed instead, since psi-im/plugins is a better location compared to psi-im/psi.

**Comment by Neustradamus** [comment]
Some tickets from psi repo must be transfered into plugins repo ^^

**Comment by Neustradamus** [comment]
@SharkyRawr: Can you confirm that the OMEMO button icon for the chat tab is enabled?

**Comment by Neustradamus** [comment]
@yessiest screenshot from issue: https://github.com/psi-im/psi/issues/616:
- https://user-images.githubusercontent.com/70776062/109429325-a3bded80-7a14-11eb-8c74-ae40876344e2.png

**Comment by lunaisnotaboy** [comment]
Yeah, this is happening for me too. I'm on macOS Monterey 12.6.2, and it's doing the same thing as the above screenshot.

**Comment by Neustradamus** [comment]
@SharkyRawr, @lunaisnotaboy: Have you tested with latest Psi+ build?
The problem is always here?

**Comment by Ri0n** [comment]
@Neustradamus I appreciate your efforts but you ping a lot of people in vain

**Comment by lunaisnotaboy** [comment]
> @SharkyRawr, @lunaisnotaboy: Have you tested with latest Psi+ build? The problem is always here?

Because of this issue, I gave up on XMPP altogether. I can try again with a new account, but it'll be a few days because of how busy I am.
